packages/notifi-react-card/lib/components/intercom/MessageGroup.tsx

```diff
@@ -34,63 +31,70 @@ export const MessageGroup: React.FC<MessageGroupProps> = ({
         classNames?.messageGroupContainer,
       )}
     >
-      {messages.map((message, index) => (
-        <div
-          className={clsx(
-            isIncoming
-              ? 'NotifiIntercomChatIncomingMessage__container'
-              : 'NotifiIntercomChatOutgoingMessage__container',
-            classNames?.messageContainer,
-          )}
-          key={index}
-        >
-          {isIncoming ? (
-            <div>
-              <img
-                height={34}
-                src={supportAvatarUrl}
-                onError={(e) => {
-                  const img = e.target as HTMLImageElement;
-                  img.style.display = 'none';
-                }}
-                className={clsx(
-                  'NotifiIntercomChatMessage__avatar',
-                  classNames?.avatar,
-                )}
-              />
-            </div>
-          ) : null}
+      {messages.map((message, index) => {
+        const participantProfile = message.conversationParticipant.profile;
+        return (
           <div
-            key={index}
             className={clsx(
               isIncoming
-                ? 'NotifiIntercomChatIncomingMessage__body'
-                : 'NotifiIntercomChatOutgoingMessage__body',
-              classNames?.messageBody,
+                ? 'NotifiIntercomChatIncomingMessage__container'
+                : 'NotifiIntercomChatOutgoingMessage__container',
+              classNames?.messageContainer,
             )}
+            key={index}
           >
             {isIncoming ? (
-              <div
-                className={clsx(
-                  'NotifiIntercomChatOutgoingMessage__sender',
-                  classNames?.sender,
-                )}
-              >
-                {message.conversationParticipant.resolvedName}
+              <div>
+                <img
+                  height={34}
```

Comment:
rather than embedding this inside a div, it would be better to use `display: block` on the img
 
also, instead of an explicit height, we should provide that in the default styles.